### DISCUSSION TITLE
Rename `aztec-gddt` to something else

### DISCUSSION THREAD

**Discussion by danlessa** [jackhack00/aztec-gddt#99] [General]
We should decide a more relatable name for what this simulation product is about

**Comment by jackhack00** [comment]
Aztec Labs has previously suggested "SimNet", which works a bit more nicely with testnet / mainnet etc nomenclature. 
I am unsure if that's ideal though, as we are not necessarily doing a "network" of models. 
In our current report I've punted this slightly by saying "our ongoing simulation efforts". 
Happy for any naming suggestions!
